crates/router/src/connector/paypal/transformers.rs

```diff
@@ -67,9 +73,21 @@ pub enum PaypalPaymentIntent {
 }
 
 #[derive(Default, Debug, Clone, Serialize, Eq, PartialEq, Deserialize)]
+pub struct Amount {
+    currency_code: storage_enums::Currency,
+    value: String,
+}
+
+#[derive(Default, Debug, Serialize, Deserialize, Eq, PartialEq)]
 pub struct OrderAmount {
     pub currency_code: storage_enums::Currency,
     pub value: String,
+    pub breakdown: AmountBreakdown,
```

Comment:
Please keep the existing struct as it is, since it is being used in webhooks and other flows. For your use case create new struct Amount and add breakdown field